aoc-2022-rust/src/days/day_13/general_remarks.md

```diff
@@ -0,0 +1,5 @@
+# General Remarks
+
+- Use clippy! it contains a lot of useful tips :)
```

Comment:
Lovely I used `cargo clippy --fix` and it did a lot. Is there some way to integrate it with neovim as well?